src/python/ddapp/tasks/robottasks.py

```diff
@@ -178,7 +178,7 @@ def getDefaultProperties(properties):
         properties.setProperty('Message', 'Plan is invalid. Do you want to accept it anyway?')
 
     def run(self):
-        if robotSystem.ikPlanner.lastManipPlan and max(robotSystem.ikPlanner.lastManipPlan.plan_info) <= 10:
+        if robotSystem.ikPlanner.lastManipPlan and max(robotSystem.ikPlanner.lastManipPlan.plan_info) <= 10 and min(robotSystem.ikPlanner.lastManipPlan.plan_info) >= 0:
```

Comment:
I'm not familiar with this, is this saying that sometimes an infeasible plan_info is < 0?